src/hipscat/catalog/catalog_info.py

```diff
@@ -0,0 +1,11 @@
+from dataclasses import dataclass
+
+from hipscat.catalog.dataset.base_catalog_info import BaseCatalogInfo
+
+
+@dataclass
+class CatalogInfo(BaseCatalogInfo):
+    """Catalog Info for a HEALPix Hive partitioned Catalog"""
+    epoch: str = "J2000"
+    ra_column: str = "ra"
+    dec_column: str = "dec"
```

Comment:
For a catalog, `catalog_name` and `epoch` (and maybe `ra_column` and `dec_column`) are required. Is it straightforward to extend `required_fields`?

Reply:
Yep! I didn't add them since they had default values, but you're right it's a good check to add.